### DISCUSSION TITLE
Zabbix environment variables

### DISCUSSION THREAD

**Discussion by june012006** [tiredofit/docker-db-backup#357] [Q&A] [answered]
Are there any environment variables to change the "ServerActive" in the zabbix config in the container?    My zabbix server is a docker container on the same server, and if possible, I'd rather not need to set up a proxy as well on the same server

**Comment by tiredofit** [comment, marked as answer]
https://github.com/tiredofit/docker-alpine?tab=readme-ov-file#zabbix-options
